lib/nailed/github.rb

```diff
@@ -1,14 +1,75 @@
+#
+# Nailed::Github
+#
 module Nailed
+  
+  #
+  # github helpers
+  #
+  def get_org_repos(github_client, org)
+    all_repos = github_client.org_repos(org)
+    all_repos.map(&:name)
+  end
+
+  def fill_db_after_migration(github_client)
```

Comment:
I was only meaning the `fill_db_after_migration` method. the other ones I fully agree